po/ka.po

```diff
@@ -0,0 +1,308 @@
+msgstr[1] ""
+"სწორი პაროლი უნდა წარმოადგენდეს დიდი და პატარა რეგისტრის ასოების, ციფრებისა "
+"და\n"
+"სხვა სიმბოლოების ნარევს.  შეგიძლიათ გამოიყენოთ მაქსიმუმ %d სიმბლოს, %d (სულ "
+"3) კლასიდან.\n"
+"დიდი რეგისტრის სიმბოლო და დამამთავრებელი ციფრი არ\n"
+"ითვლება გამოყენებული კლასების რაოდენობაში.\n"
+
+#: pam_passwdqc.c:141
+#, c-format
+msgid ""
+"A passphrase should be of at least %d word, %d to %d characters long, and\n"
+"contain enough different characters.\n"
+msgid_plural ""
+"A passphrase should be of at least %d words, %d to %d characters long, and\n"
+"contain enough different characters.\n"
+msgstr[0] ""
+"საკვანძო სიტყვა მინიმუმ %d სიტყვას უნდა შეიცავდეს, იყოს %d-%d\n"
+"სიმბოლოს სიგრძის და შეიცავდეს საკმარის განსხვავებულ სიმბოლოებს.\n"
+msgstr[1] ""
+"საკვანძო სიტყვა მინიმუმ %d სიტყვას უნდა შეიცავდეს, იყოს %d-%d\n"
+"სიმბოლოს სიგრძის და შეიცავდეს საკმარის განსხვავებულ სიმბოლოებს.\n"
+
+#: pam_passwdqc.c:149
+#, c-format
+msgid ""
+"Alternatively, if no one else can see your terminal now, you can pick this "
+"as\n"
+"your password: \"%s\".\n"
+msgstr ""
+"ან, თუ თქვენს ტერმინალს ამ წამს არავინ უყურებს, პაროლად \n"
+"\"%s\" შეგიძლიათ აიღოთ.\n"
+
+#: pam_passwdqc.c:152
+#, c-format
+msgid ""
+"This system is configured to permit randomly generated passwords only.\n"
+"If no one else can see your terminal now, you can pick this as your\n"
+"password: \"%s\".  Otherwise come back later.\n"
+msgstr ""
+"სისტემა მორგებულია, რომ მხოლოდ შემთხვევითი პაროლებით შეგიშვათ.\n"
+"თუ თქვენს ტერმინალს ამ წამს არავინ უყურებს, პაროლად შეგიძლიათ\n"
+"\"%s\" აიღოთ.  წინააღმდეგ შემთხვევაში, მოგვიანებით დაბრუნდით.\n"
+
+#: pam_passwdqc.c:156
+msgid ""
+"This system is configured to use randomly generated passwords only,\n"
+"but the attempt to generate a password has failed.  This could happen\n"
+"for a number of reasons: you could have requested an impossible password\n"
+"length, or the access to kernel random number pool could have failed."
+msgstr ""
+"სისტემა მხოლოდ შემთხევითი პაროლების გამოყენებაზეა მორგებული,\n"
+"მაგრამ პაროლის გენერაციის მცდელობა წარუმატებლად დასრულდა.  ეს რამდენიმე "
+"მიზეზით\n"
+"შეიძლება მოხდეს: ან მოთხოვნილი პაროლის სიგრძე მიუღებელია, ან\n"
+"შემთხვევითი რიცხვების გენერატორთან წვდომა წარუმატებლად დასრულდა."
+
+#: pam_passwdqc.c:161
+msgid "This password may be too long for some services.  Choose another."
+msgstr ""
+"ეს პაროლი ზოგიერთი სერვისისთვის შეიძლება ძალიან გრძელია.  სხვა აირჩიეთ."
+
+#: pam_passwdqc.c:163
+msgid "Warning: your longer password will be truncated to 8 characters."
+msgstr "გაფრთხილება: თქვენი გრძელი პაროლი 8 სიმბოლომდე შეიკვეცება."
+
+#: pam_passwdqc.c:165
+#, c-format
+msgid "Weak password: %s."
+msgstr "სუსტი პაროლი: %s."
+
+#: pam_passwdqc.c:167
+msgid "Sorry, you've mistyped the password that was generated for you."
+msgstr "უკაცრავად, თქვენთვის დაგენერირებული პაროლი არასწორად აკრიფეთ."
+
+#: pam_passwdqc.c:169
+msgid "Sorry, passwords do not match."
+msgstr "პაროლები არ ემთხვევა."
```

Comment:
Apparently, `Sorry` is present in msgid but is not present in the translated msgstr.